doc/theory/sh_basis.rst

```diff
@@ -27,26 +35,29 @@ computed using the expression:
     a_l^m = \int_S f(\theta, \phi) Y_l^m(\theta, \phi) ds
 
 Once the coefficients are computed, the function $f(\theta, \phi)$ can be
-approximately computed as:
+computed as:
 
 ..  math::
 
     f(\theta, \phi) = \sum_{l = 0}^{\inf} \sum_{m = -l}^{l} a^m_l Y_l^m(\theta, \phi)
```

Comment:
Having a look at the rendered page, looks like `\inf` is a typo and should be `\infty`. @CHrlS98 given that the file is being modified in this PR, do you mind changing this, please?

Reply:
No problem :+1:

Reply:
Fixed in d928b33. Thanks @CHrlS98 !